### ISSUE TITLE
Narrator does not announce the column header in Calendar

### ISSUE THREAD

**Issue by DhirajKhot** [OfficeDev/office-ui-fabric-react#8078]
### Environment Information 
 
- __Package version(s)__:  6.129

### Please provide a reproduction of the bug in a codepen:
 
Navigate to https://developer.microsoft.com/en-us/fabric#/components/Calendar
Check for any Calendar. 
Use the Narrator and check the announcements for the dates. 

 
#### Actual behavior:
 
Narrator does not announce the column header. 
 
#### Expected behavior:
 
Narrator should announce the column header. This will help the Narrator users to get the idea of DAY for the current date.
 
MAS Reference: [MAS 4.2.3 - Row, Column, and Headers](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B30e4fb40-dcc1-45b4-b714-5e2c28b23ae8%7D)

**Comment by DhirajKhot** [comment]
We have new issues with this fix. 
The current date is announced as editable - which is wrong. 
For other dates Narrator is announcing the column header but the dates are not announced. 
It is announcing some irrelevant info - no current char. 

![image](https://user-images.githubusercontent.com/20378231/53863683-cd540200-4010-11e9-8792-6f869275bf44.png)

**Comment by aneeshack4** [comment]
@lorejoh12 can you take a look at this new issue that @DhirajKhot has found? I'm off shield and onto other work now so I don't have the bandwidth for any more work on Calendar.

**Comment by DhirajKhot** [comment]
@lorejoh12 Can we reopen this bug to track the progress?
